modules/libcom/src/osi/os/RTEMS-score/osdEvent.c

```diff
@@ -48,11 +49,9 @@ epicsEventCreate(epicsEventInitialState initialState)
     rtems_status_code sc;
     rtems_id sid;
     rtems_interrupt_level level;
```

Comment:
Is this used?

Reply:
No, `level` is no longer used and should be removed.